holoviews/operation/stats.py

```diff
@@ -0,0 +1,135 @@
+import param
+import numpy as np
+
+from ..core import Dimension, Dataset, NdOverlay
+from ..core.dimension import Dimension
+from ..core.operation import Operation
+from ..core.options import Compositor, Store, Options, StoreOptions
+from ..core.util import basestring, find_minmax, cartesian_product
+from ..element import Curve, Area, Image, Polygons
+
+from .element import contours
+
+
+class univariate_kde(Operation):
```

Comment:
Will need a docstring. I know you were probably already planning to add one - just making sure!

Reply:
Done.